src/utils/isAfterDay.js

```diff
@@ -0,0 +1,18 @@
+import moment from 'moment';
+
+export default function isAfterDay(a, b) {
```

Comment:
we could also probs update usage to only have one. I just thought it was a bit clear to use one over the other in some cases.